code/style.css

```diff
@@ -1,4 +1,312 @@
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.main-text-middle {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.main-text-bottom {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+h2 {
+  color: #13233d;
+}
+
+p {
+  font-weight: lighter;
+}
+
+/* The other news */
+.other-news {
+  display: flex;
+  flex-direction: column;
+}
+
+.article-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.news-article {
+  display: flex;
+  flex-direction: column;
+}
+.other-image{
+  display: flex;
+  flex-direction: column; 
+}
+
+.article-text {
+  padding: 15px;
+}
+
+.text-top {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.text-middle {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.text-bottom {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+/* Footer with social media */
+#lab_social_icon_footer {
+  padding: 15px 0;
+  background-color: #13233d;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#lab_social_icon_footer a {
+  color: #f3e8dd;
+} 
+
+/* Tablet */
+@media all and (min-width: 668px) {
+  
+  /* Logo & Navbar */
+  .menu {
+    align-items: center;     
+  }
+
+  a:hover {
+    color: #f3e8dd;
+  }
+
+  .logo {
+    flex: 1;
+  }
+
+  /* The big news content */ 
+  .columns {
+    display: flex;
+    flex-direction: row;
+    padding: 25px;
+  }
+  
+  .column {
+    display: flex;
+  }
+  
+  .image {
+    flex: 1;
+  }
+
+  .main-img {
+    max-width: 100%;
+    border: solid 10px #ffafaf;
+
+  }
+  
+  .article {
+    flex: 1; 
+    margin-left: 10px;
+  }
+
+
+  /* The other news */
+  .other-news {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .article-box {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .other-news-image {
+    flex: 2;
+    max-width: 100%;
+    padding: 20px;
+  }
+
+  .news-article {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    margin: 0px;
+    border: solid 5px #13233d;
+  }
+
+  .article-text {
+    flex: 1;
+    padding: 15px;
+  } 
+}
+
+/* Desktop */
+@media all and (min-width: 1024px) {
+  
+   /* Logo & Navbar */
+  .menu {
+    align-items: center;     
+    flex-wrap: nowrap;
+    background: none;
+  }
+
+  .logo {
+    order: 0;
+  }
+
+  .item {
+    order: 1;
+    position: relative;
+    display: block; 
+    width: auto;
+  }
+
+  /* The big news content */
+  .columns {
+    display: flex;
+    flex-direction: row;
+    padding: 30px;
+  }
+  
+  .column {
+    display: flex;
+    justify-content: center;
+  }
+  
+  .image {
+    flex: 2;
+    align-items: center;
+  }
+
+  .main-img {
+    max-width: 90%;
+  }
+  
+  .article {
+    flex: 2; 
+    margin-left: 10px;
+  }
+
+  h2 {
+    font-size: 25px;
+  }
+
+
+  /* The other news */
+  .other-news {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+  }
+
+  .article-box {
+    display: flex;
+    flex-direction: row;
+  }
+    
+  .news-article img:hover{
```

Comment:
What does this styling do? I can´t see the hover-effect, is it something broken in the code?